cloud-stream-batch-consume/src/main/resources/application.yaml

```diff
@@ -0,0 +1,28 @@
+spring:
+  cloud:
+    function:
+      definition: batchConsume
+    stream:
+      bindings:
+        batchConsume-in-0:
+          destination: 'batch/consume/topic'
+          group: batch
+          content-type: 'application/octet-stream'
```

Comment:
suggest removing.

Reply:
I am assuming you are referring to the removal of content-type - the others have to stay, right!

Removed the content-type property (last conversation) as well.

Reply:
correct, I mean `content-type`